generate-manifest.js

```diff
@@ -0,0 +1,58 @@
+const fs = require("fs");
+const path = require("path");
+
+const FILE_EXTENSION = ".kcl";
+const MANIFEST_FILE = "manifest.json";
+const COMMENT_PREFIX = "//";
+
+// Function to read and parse .kcl files
+const getKclMetadata = (filePath) => {
+  const content = fs.readFileSync(filePath, "utf-8");
+  const lines = content.split("\n");
+
+  if (lines.length < 2) {
+    return null;
+  }
+
+  const title = lines[0].replace(COMMENT_PREFIX, "").trim();
+  const description = lines[1].replace(COMMENT_PREFIX, "").trim();
```

Comment:
Oh right, just the first line, I guess if there's more, you only need the first one for the cmd bar anyway.